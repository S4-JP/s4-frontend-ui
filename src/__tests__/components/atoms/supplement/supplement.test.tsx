import React from 'react'
import { render } from '@testing-library/react'
import { Supplement } from '../../../../index'

describe('Supplementのレンダリング確認', () => {
  test('propsに設定した値でレンダリングされる。', () => {
    const { getByTestId } = render(<Supplement text="test" />)

    expect(getByTestId('atoms-sup-text')).toMatchSnapshot()
  })
})
