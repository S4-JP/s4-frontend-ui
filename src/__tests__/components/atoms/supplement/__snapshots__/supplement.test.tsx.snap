// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Supplementのレンダリング確認 propsに設定した値でレンダリングされる。 1`] = `
<p
  class="sc-dsQDmV jZpZSg"
  data-testid="atoms-sup-text"
>
  test
</p>
`;
